### DISCUSSION TITLE
Cannot get Fan Control to launch

### DISCUSSION THREAD

**Discussion by Phax3** [Rem0o/FanControl.Releases#1737] [Q&A]
Every Time I try to run the program I get an error and then it says to check the log file. I have attached the log file, thank you in advance.
System Info: Razer Blade Stealth Laptop, Sonnet Breakaway box egpu with a gtx 1060
[log.txt](https://github.com/Rem0o/FanControl.Releases/files/11336267/log.txt)

**Comment by Rem0o** [comment]
Fan Control is not supported on laptops.

**Reply by KurtSmart** [reply]
Will there be plugins being released to enable support? I have a MSI laptop and loved the software but now it doesn't work anymore on my laptop
